OrbitGl/DataManager.h

```diff
@@ -57,7 +57,7 @@ class DataManager final {
   void set_user_defined_capture_data(const UserDefinedCaptureData& user_defined_capture_data) {
     user_defined_capture_data_ = user_defined_capture_data;
   }
-  [[nodiscard]] const UserDefinedCaptureData& user_defined_capture_data() const {
+  [[nodiscard]] const UserDefinedCaptureData& mutable_user_defined_capture_data() const {
```

Comment:
Super nit: Following the style guide, the only exception for the PascalCase in function names would be the getter/setter with exactly the name of the field (+`set_`). So I guess `GetMutableUserDefinedCaptureData` would be better here.

Reply:
It's not exactly worded like that, if I read the style guide right. I think one can make the case that +mutable_ is fine as well. 

(But I should actually change that in App for GetCaptureData()).

Reply:
Actually, no, GetCaptureData() in App is getting capture_data.value(), so it should not follow this rule.

Reply:
Agree.